clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookURI.ts

```diff
@@ -120,7 +118,13 @@ export class CopybookURI {
       result.forEach(
         (value, index) =>
           (result[index] =
-            C4Z_FOLDER + "/" + COPYBOOKS_FOLDER + "/" + profile + "/" + value),
+            Utils.getExtensionFolder() +
+            ZOWE_FOLDER +
+            COPYBOOKS_FOLDER +
+            "/" +
+            profile +
+            "/" +
+            value),
```

Comment:
I think there is `/` missing, also use appropriate join function.

Reply:
resolved at : 224e9b8e00a6753439dd54d8da5d6df093b6cf89